mkt/games/views.py

```diff
@@ -0,0 +1,25 @@
+from mkt.fireplace.views import MultiSearchView
+from mkt.games.filters import DailyGamesFilter
+from mkt.games.paginator import ESGameAggregationPaginator
+from mkt.games.serializers import (GamesESAppSerializer,
+                                   GamesESWebsiteSerializer)
+from mkt.search.filters import DeviceTypeFilter, PublicAppsFilter
+from mkt.search.serializers import DynamicSearchSerializer
+
+
+class DailyGamesView(MultiSearchView):
+    authentication_classes = []
+    cors_allowed_methods = ['get']
+    filter_backends = [PublicAppsFilter, DeviceTypeFilter, DailyGamesFilter]
+    permission_classes = []
+    paginator_class = ESGameAggregationPaginator
+    serializer_class = DynamicSearchSerializer
```

Comment:
You don't need to redefine `authentication_classes`, `cors_allowed_methods`, `permission_classes` or `serializer_class`